services/blockchain-connector/methods/formatter.js

```diff
@@ -13,19 +13,14 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const dataService = require('./dataService');
-const regex = require('./regex');
+const { formatTransaction } = require('../shared/sdk');
 
-let chainID;
-
-const isMainchain = async () => {
-	if (!chainID) {
-		const networkStatus = (await dataService.getNetworkStatus()).data;
-		chainID = networkStatus.chainID;
-	}
-	return regex.MAINCHAIN_ID.test(chainID);
-};
-
-module.exports = {
-	isMainchain,
-};
+module.exports = [
+	{
+		name: 'formatTransaction',
+		params: {
+			transaction: { optional: false, type: 'any' },
```

Comment:
maybe we can use multi type validation instead?
https://github.com/icebob/fastest-validator#multi

Reply:
Let's keep it as it for now. We already have gateway validation for transaction in place and any new improvements can be done in separate issue.